src/net/data_stream.rs

```diff
@@ -125,22 +129,74 @@ impl BlockGenerator {
     }
 }
 
+/// # Stream Packet
 /// Represents a single UDP packet sent by the stream.
 ///
-/// # Packet Format
-/// All data is sent in network-endian format. The format is as follows
+/// A "batch" of data is defined to be the data collected for a single invocation of the DSP
+/// routine. A packet is composed of as many sequential batches as can fit.
+///
+/// The packet is given a header indicating the starting batch sequence number and the number of
+/// batches present. If the UDP transmitter encounters a non-sequential batch, it does not enqueue
+/// it into the packet and instead transmits any staged data. The non-sequential batch is then
+/// transmitted in a new UDP packet. This method allows a receiver to detect dropped batches (e.g.
+/// due to processing overhead).
+///
+/// ## Data Format
+///
+/// Data sent via UDP is sent in "blocks". Each block is a single batch of ADC/DAC codes from an
+/// individual DSP processing routine. Each block is assigned a unique 16-bit identifier. The identifier
```

Comment:
Call it `sequence offset` instead of `identifier`.
Then again, let's not invest too much into these docs given that #385 will change it quite a bit.

Reply:
Not touching this for now, as this is intended to change.